src/NzbDrone.Core/Datastore/TableMapping.cs

```diff
@@ -119,40 +120,40 @@ public static void Map()
             Mapper.Entity<Track>().RegisterModel("Tracks")
                 .Ignore(t => t.HasFile)
                 .Ignore(t => t.AlbumId)
-                .Ignore(t => t.ArtistId)
                 .Ignore(t => t.Album)
                 .Relationship()
                 .HasOne(track => track.AlbumRelease, track => track.AlbumReleaseId)
                 .HasOne(track => track.ArtistMetadata, track => track.ArtistMetadataId)
-                .HasOne(track => track.TrackFile, track => track.TrackFileId)
+                .For(track => track.TrackFile)
+                .LazyLoad(condition: track => track.TrackFileId > 0,
+                          query: (db, track) => db.Query<TrackFile>()
+                          .Join<TrackFile, Album>(JoinType.Inner, t => t.Album, (t, a) => t.AlbumId == a.Id)
+                          .Join<TrackFile, Artist>(JoinType.Inner, t => t.Artist, (t, a) => t.Album.Value.ArtistMetadataId == a.ArtistMetadataId)
+                          .Join<Artist, ArtistMetadata>(JoinType.Inner, a => a.Metadata, (a, m) => a.ArtistMetadataId == m.Id)
+                          .Where(t => t.Id == track.TrackFileId)
+                          .SingleOrDefault())
                 .For(t => t.Artist)
-                .LazyLoad(condition: t => t.AlbumReleaseId > 0, query: (db, t) => db.Query<Artist>().QueryText(string.Format(
-                   "/* LazyLoading Artist for Track */\n" +
-                   "SELECT Artists.* " +
-                   "FROM Artists " +
-                   "JOIN Albums ON Albums.ArtistMetadataId = Artists.ArtistMetadataId " +
-                   "JOIN AlbumReleases ON AlbumReleases.AlbumId = Albums.Id " +
-                   "WHERE AlbumReleases.Id = {0}",
-                   t.AlbumReleaseId)).SingleOrDefault());
+                .LazyLoad(condition: t => t.AlbumReleaseId > 0, query: (db, t) => db.Query<Artist>()
+                          .Join<Artist, ArtistMetadata>(JoinType.Inner, a => a.Metadata, (a, m) => a.ArtistMetadataId == m.Id)
```

Comment:
In my testing this made the import quite a bit faster.  They get converted into sqlite joins (which look almost identical to the old queries when I turned on trace query logging) by the mapper but pull back more columns so that the joined objects are all populated instead of needing extra lazyload calls to the database.  I think this really helped with the slowdown @Qstick was seeing